packages/f/filament/xmake.lua

```diff
@@ -6,7 +6,8 @@ package("filament")
 
     if is_plat("windows") and is_arch("x64") then
         add_urls("https://github.com/google/filament/releases/download/v$(version)/filament-v$(version)-windows.tgz")
-        add_versions("1.9.23", "8f5728dd944c052f991bfb15d18cd6fc8f678d0cbd37bb066d76bf682ab789c8")
+        add_versions("1.49.3", "f465e45033b928a92e5b4a5a3b6534022f5428bfdb8016c953204f63842ac55c")
+    add_versions("1.9.23", "8f5728dd944c052f991bfb15d18cd6fc8f678d0cbd37bb066d76bf682ab789c8")
```

Comment:
@waruqi I think we should detect when add_versions are behind conditions and disable auto-update, what do you think?

Reply:
I have improved it.